tests/upgrades/test_bookmarks.py

```diff
@@ -21,136 +21,137 @@
 from robottelo.constants import BOOKMARK_ENTITIES
 
 
-@pre_upgrade
-def test_pre_create_public_disable_bookmark(request):
-    """Create public disable bookmarks for system entities using available bookmark
-    data.
-
-    :id: c4f90034-ea57-4a4d-9b73-0f57f824d89e
-
-    :Steps:
-
-        1. Create public disable bookmarks before the upgrade for all system entities
-        using available bookmark data.
-        2. Check the bookmark attribute status(controller, name, query public)
-        for all the system entities.
-
-    :expectedresults: Public disabled bookmark should be created successfully
-
-    :BZ: 1833264, 1826734, 1862119
-
-    :CaseImportance: Critical
+class TestPublicDisableBookmark:
+    """
+    Created Public disable Bookmarks in pre upgrade should be unchanged after post upgrade
```

Comment:
```suggestion
    Created public disabled Bookmarks in pre upgrade should be unchanged post upgrade
```
"created" and "disabled" both need to be past tense. Either say "post upgrade" or "after upgrade".

Reply:
Fixed.